src/GameOverview.tsx

```diff
@@ -4,41 +4,42 @@ import Grid from '@mui/material/Grid';
 import CapitalismIcon from '@mui/icons-material/Whatshot';
 import SocialismIcon from '@mui/icons-material/AcUnit';
 import TurnIcon from '@mui/icons-material/HourglassEmpty';
+// @ts-ignore
 import WarIcon from './img/rifle.svg?react';
+// @ts-ignore
```

Comment:
Instead of ignoring these how do you feel about wrapping these SVGs into their own tsx components?  Its a small uplift and adds to the overall type safety while not ignoring anything.  I whipped up a little example of it if you want to see what I mean.